clang/include/clang/AST/Stmt.h

```diff
@@ -649,6 +649,8 @@ class alignas(void *) Stmt {
     /// It is 0 otherwise.
     LLVM_PREFERRED_TYPE(bool)
     unsigned HasFPFeatures : 1;
+    LLVM_PREFERRED_TYPE(bool)
```

Comment:
Nit: Newline and maybe a comment?

Reply:
Good idea, see https://github.com/llvm/llvm-project/pull/100272/commits/e8afeed6dfc5d427b83d52d69c8f92b27fd1220e

thanks